File: 프로그래머스/src/프로그래머스_02262022/두_정수_사이의_합.cpp
#include <string>
#include <vector>

using namespace std;

long long solution(int a, int b)
{
    long long answer = 0;
    if (a == b)
        return a;
    long long start = a > b ? b : a;
    long long end = a > b ? a : b;
    for (long long i = start; i <= end; i++)
        answer += i;
    return answer;
}

File: 2026/Level1/두정수사이의합.py
# https://school.programmers.co.kr/learn/courses/30/lessons/12912?language=python3
def swap(a,b):
    return (b,a)

def solution(a, b):
    answer = 0
    if a == b:
        return a
    if b<a :
        a,b = swap(a,b)
    
    for x in range(a,b+1):
        answer += x
    return answer